backend/app/api/endpoints/adapter/executors.py

```
@@ -67,3 +69,53 @@ async def update_subtask(
+    data: Dict[str, Any] = Field(..., description="Chunk data payload")
+    executor_name: Optional[str] = Field(default=None, description="Executor name")
+    executor_namespace: Optional[str] = Field(
+        default=None, description="Executor namespace"
+    )
+    timestamp: Optional[str] = Field(default=None, description="ISO timestamp")
+
+
+@router.post("/tasks/chunk")
+async def receive_chunk(
+    request: ChunkUpdateRequest,
+    db: Session = Depends(get_db),
```

Ruff (0.14.10)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)